src/models/visits.js

```diff
@@ -0,0 +1,55 @@
+import mongoose from 'mongoose';
+
+const visitSchema = new mongoose.Schema({
+    visitor: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Visitor',
+        required: true
+    },
+    employee: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'User',
+        required: true
+    },
+    entryTime: {
+        type: Date,
+        default: Date.now,
+        required: true
+    },
+    exitTime: {
+        type: Date,
+        default: null
+    },
+    duration: {
+        type: String,
+        required: true
+    },
+    purpose: {
+        type: String,
+        trim: true,
+        required: true
+    },
+    relatedTo: {
+        type: mongoose.Schema.Types.ObjectId,
```

Comment:
can we create common const for mongoose.Schema.Types.ObjectId

Reply:
for that now created a variable